controllers/datamovement_controller.go

```diff
@@ -84,6 +104,10 @@ func newInvalidError(format string, a ...any) *invalidError {
 func (i *invalidError) Error() string { return i.err.Error() }
 func (i *invalidError) Unwrap() error { return i.err }
 
+// Regex to scrape the progress output of the `dcp` command. Example output:
+// Copied 1.000 GiB (10%) in 1.001 secs (4.174 GiB/s) 9 secs left ..."
+var progressRe = regexp.MustCompile(`Copied\s(\d+\.\d+)\s(\S{3})\s\((\d{2,3})%\)\sin`)
```

Comment:
The order is usually Constants, Variables, Types & Functions. So I'd move this up after the constants